src/models/UserMapper.php

```diff
@@ -227,6 +228,39 @@ public function isSiteAdmin($user_id)
         return false;
     }
 
+    /**
+     * Check if the user represented by $user_id is trusted
+     *
+     * @param $user_id
+     * @return bool
+     */
+    public function isTrusted($user_id)
+    {
+        $results = $this->getUsers(1, 0, 'user.ID=' . (int) $user_id, null);
+        if (isset($results[0]) && $results[0]['trusted'] == 1) {
+            return true;
+        }
+
+        return false;
+    }
+
+    public function setTrustedStatus($trustedStatus, $user_id)
+    {
+        $verify_sql = "update user set trusted = :trusted_status "
+            . "where ID = :user_id";
+
+        $verify_stmt = $this->_db->prepare($verify_sql);
+        $verify_data = array("trusted_status" => (int) $trustedStatus, "user_id" => $user_id);
+
+        $result = $verify_stmt->execute($verify_data);
+
+        if ($result) {
```

Comment:
As [PDO_Statement::execute()](http://php.net/manual/de/pdostatement.execute.php) always returns a boolean, wouldnt ```return $verify_stmt->execute($verify_data);``` be enough?